moviesite/polls/movie_api.py

```diff
@@ -0,0 +1,60 @@
+from mechanize import Browser
+from BeautifulSoup import BeautifulSoup
+import sys
+import re
+import json
```

Comment:
please go through this section to learn about how you should arrange imports - https://www.python.org/dev/peps/pep-0008/#imports